### DISCUSSION TITLE
Formulating a Degree Schema

### DISCUSSION THREAD

**Discussion by charliemahana** [UTDNebula/nebula-api#46] [Ideas]
# Formulating a Degree Schema
This GitHub discussion is meant to serve as a thread specifically for documenting the design of the degree schema.

**Comment by jmyrick02** [comment]
A degree should be a collection with the 'type' set to "Degree" containing all the courses needed to fulfill a degree using nested collections. A proposed collection schema can be found in #54.

**Reply by charliemahana** [reply]
I know we've already spoken about this idea so if the rest of team approves of it, I think that this could work quite well.
